config/proguard/okhttp-proguard-rules.pro

```diff
@@ -10,3 +10,12 @@
 -dontwarn okhttp3.Authenticator*, okhttp3.CookieJar*, okhttp3.Dns*
 -dontwarn okhttp3.internal.http2.PushObserver*, okhttp3.internal.io.FileSystem*
 -dontwarn org.conscrypt.Conscrypt*
+-dontwarn org.bouncycastle.jsse.BCSSLSocket
+-dontwarn org.bouncycastle.jsse.BCSSLParameters
+-dontwarn org.bouncycastle.jsse.provider.BouncyCastleJsseProvider
+-dontwarn org.bouncycastle.provider.BouncyCastleJsseProvider
+-dontwarn org.bouncycastle.provider.BouncyCastleJsseProvider
+-dontwarn org.openjsse.javax.net.ssl.SSLSocket
+-dontwarn org.openjsse.javax.net.ssl.SSLParameters
+-dontwarn org.openjsse.net.ssl.OpenJSSE
+-dontwarn org.openjsse.net.ssl.OpenJSSE
```

Comment:
Do we know why these changed with API 30? I'm assuming it's tied to the OkHttp version.

What would be nice context to get is:
- The changelog, issue, commit, etc. for OkHttp's latest version that points to these being okay exceptions to add
- The changelog, issue, commit, etc. for why the older version of OkHttp isn't compatible with targeting SDK 30

Reply:
Added link

Reply:
Thanks, that helps.